client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/content/get-titan-footer-details.ts

```diff
@@ -0,0 +1,63 @@
+import { localizeUrl } from '@automattic/i18n-utils';
+import { translate } from 'i18n-calypso';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { TITAN_CONTROL_PANEL_CONTEXT_GET_MOBILE_APP } from 'calypso/lib/titan/constants';
+import { getTitanControlPanelRedirectPath } from 'calypso/my-sites/email/paths';
+import type { ThankYouFooterDetailProps } from 'calypso/components/thank-you-v2/footer';
+
+export default function getTitanFooterDetails(
```

Comment:
The Titan footer details is taken from https://github.com/Automattic/wp-calypso/pull/88390/files#diff-4de648f48d7bc4d2c6c589e290223d57cea6d434906113c36e52cc88b2877c0fL35. The idea is to allow the same content to be shared by `<TitanSetUpThankYou>` and `<GenericThankYou>`.